sci-geosciences/gmt/gmt-4.5.18-r1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="https://gmt.soest.hawaii.edu/"
+SRC_URI="
+	mirror://gmt/${P}-src.tar.bz2
+	gmttria? ( mirror://gmt/${P}-non-gpl-src.tar.bz2 )"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples debug +gdal gmttria +gshhg htmldoc +metric mex +netcdf octave postscript tutorial"
+
+RDEPEND="
+	!sci-biology/probcons
+	gdal? ( sci-libs/gdal )
+	gshhg? ( sci-geosciences/gshhg-data )
+	netcdf? ( >=sci-libs/netcdf-4.1 )
+	octave? ( sci-mathematics/octave )"
+DEPEND="${RDEPEND}"
+
+# mex can use matlab too which i can't test
+REQUIRED_USE="
+	mex? ( octave )
+"
+
+# hand written make files that are not parallel safe
+MAKEOPTS+=" -j1"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.5.9-no-strip.patch
+	"${FILESDIR}"/${PN}-4.5.6-respect-ldflags.patch
+	)
+
+AUTOTOOLS_IN_SOURCE_BUILD=1
+
+src_configure() {
+	econf \
+		--libdir=/usr/$(get_libdir)/${P} \
+		--includedir=/usr/include/${P} \
+		--datadir=/usr/share/${P} \
+		--docdir=/usr/share/doc/${P} \
+		--disable-update \
+		--disable-matlab \
+		--disable-xgrid \
+		--disable-debug \
+		$(use_enable gdal) \
+		$(use_enable netcdf) \
+		$(use_enable octave) \
+		$(use_enable debug devdebug) \
+		$(use_enable !metric US) \
+		$(use_enable postscript eps) \
+		$(use_enable mex) \
+		$(use_enable gmttria triangle) \
+		$(use_with gshhg gshhg-dir /usr/share/gshhg)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install-all
+	einstalldocs
+
+	# Remove various documentation
+	use doc || rm -rf "${ED}/usr/share/doc/${PF}/pdf" || die
```

Comment:
Use full `if ...`, this is horribly unreadable.

Reply:
Yes, mix of || and if. Corrected.